src/executor/Scheduler.ts

```diff
@@ -315,6 +350,10 @@ export class Scheduler {
             case 'css':
                 jobTracker = this.compressCssJobs;
                 break;
+            case 'yaml':
+                watchDir = path.join(plugin.pluginDir, 'api');
+                jobTracker = this.yamlJobs;
```

Comment:
Maybe "yaml" as the type is too generic, because we are specifically compiling API descriptions that happen to be yaml files. But since yaml is not directly bound to that purpose, and we have yaml files with other purposes as well (that don't need any compilation), we should specify the type more narrowly. E.g. as "yamlApi", like we differentiated "tsE2E" from "ts".